cmd/kubeapps-apis/core/packages/v1alpha1/repositories.go

```diff
@@ -7,6 +7,8 @@ import (
 	"context"
 	"fmt"
 
+	. "github.com/ahmetb/go-linq/v3"
```

Comment:
You like controversy Greg :P https://github.com/golang/go/issues/29326 . I see you're using it for the LINQ DSL here. Fine either way (I don't expect support for dot imports will get removed, and if it does, it'll be easy to update).

Reply:
not particularly. Do you? https://github.com/kubeapps/kubeapps/blob/c6b77631c917a6eed5afc138a17f95f9fcd75618/cmd/kubeapps-apis/core/packages/v1alpha1/packages.go#L10
Just copied your code 😛

Reply:
Haha - not me, but I did review it there too it seems :)